authzed/api/v1/permission_service.proto

```diff
@@ -349,6 +360,46 @@ message CheckPermissionResponse {
   PartialCaveatInfo partial_caveat_info = 3 [ (validate.rules).message.required = false ];
 }
 
+
+message CheckPermissionsBulkRequest {
```

Comment:
add documentation on the request itself, including that order is maintained, we optimize batches of the same permission for the same subject, etc

Reply:
Done